services/sso/src/main/java/software.amazon.awssdk.services.sso/internal/ExpiredTokenException.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.sso.internal;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import software.amazon.awssdk.annotations.SdkInternalApi;
+import software.amazon.awssdk.core.SdkField;
+import software.amazon.awssdk.core.SdkPojo;
+import software.amazon.awssdk.core.exception.SdkClientException;
+
+/**
+ * <p>
+ * The session token that was passed is expired or is not valid.
+ * </p>
+ */
+@SdkInternalApi
+public class ExpiredTokenException extends SdkClientException {
```

Comment:
If customers will see this, it'll have to be a public API. Should we just use a generic SdkClientException, or do you think customers will want to catch this exception specifically and do something about it?

Reply:
It is mentioned in the SEP that this exception will be thrown. I guess it's safer for us to use the same exception as the SEP so the users wouldn't get confused? I can change it to public though.

Reply:
I suppose someone might want to "catch" ExpiredTokenException and update their credentials. I guess it's fine to add a new public class here, then.